magellan/sampler/down_sample.py

```diff
@@ -0,0 +1,197 @@
+# coding=utf-8
+"""
+This module contains sampling related routines
+"""
+import os
+
+from random import randint
+import math
+import logging
+import random
+import numpy as np
+import pandas as pd
+import pyprind
+from magellan.utils.generic_helper import get_install_path
+import magellan.catalog.catalog_manager as cm
+
+LOGGER = logging.getLogger(__name__)
+INSTALL_PATH = get_install_path()
```

Comment:
We will avoid global varaibles in general. You can call this function (get_install_path) where you use this global variable.